connector/rest/src/test/java/it/pagopa/selfcare/dashboard/connector/rest/UserConnectorImplTest.java

```diff
@@ -37,18 +38,21 @@ class UserConnectorImplTest {
 
     UserConnectorImpl userConnector;
 
+    @Spy
+    UserMapper userMapper = new UserMapperImpl();
+
 
     @Test
     void getUserProductsNotFound() {
-        userConnector = new UserConnectorImpl(userApiRestClient, new InstitutionMapperImpl());
```

Comment:
In order to avoid the same constructor in all methods, we could do something like:
` @Before
    public void init() {
        LOG.info("startup");
        userConnector = new UserConnectorImpl(userApiRestClient, new InstitutionMapperImpl(), userMapper);
    }
`